src/utils/notifications.ts

```diff
@@ -16,6 +16,7 @@ export enum NotificationCategory {
   Mention = 'Mention',
   Payment = 'Payment',
   Admin = 'Admin',
+  Extension = 'Extension',
```

Comment:
The categories should be limited to only Mention, Payment and Admin, so all the extension notifications can be under Admin category. Sorry, I should really have left a comment about this when making this type.